test/integration/auth.spec.ts

```diff
@@ -506,6 +505,268 @@ describe('admin.auth', () => {
         });
     });
 
+    // Sanity check user management + email link generation + custom attribute APIs.
+    // TODO: Confirm behavior in client SDK when it starts supporting it.
+    describe('supports user management, email link generation, custom attribute and token revocation APIs', () => {
+      let tenantAwareAuth: admin.auth.TenantAwareAuth;
+      let createdUserUid: string;
+      let currentTime: number;
+      const newUserData = clone(mockUserData);
+      newUserData.email = generateRandomString(20) + '@example.com';
+      newUserData.phoneNumber = testPhoneNumber;
+      const importOptions: any = {
+        hash: {
+          algorithm: 'HMAC_SHA256',
+          key: Buffer.from('secret'),
+        },
+      };
+      const rawPassword = 'password';
+      const rawSalt = 'NaCl';
+
+      before(() => {
+        tenantAwareAuth = admin.auth().forTenant(createdTenantId);
```

Comment:
Can we do this outside `before`? Something like:

```
const tenantAwareAuth = admin.auth().forTenant(createdTenantId);
```

Here and in other similar describe blocks.

Reply:
I can't because `initializeApp` has to be called before I do this. We call it in `before` at setup: https://github.com/firebase/firebase-admin-node/blob/auth-multi-tenancy/test/integration/setup.ts#L40

It is not worth a refactor.